### DISCUSSION TITLE
vite4  + xgplayer-hls.js  会报错 是兼容问题么

### DISCUSSION THREAD

**Discussion by Eveningbreezes** [bytedance/xgplayer#1062] [Q&A]
![image](https://github.com/bytedance/xgplayer/assets/72544131/1aa73ff5-83ff-4a8c-8db0-d552d523269f)
![image](https://github.com/bytedance/xgplayer/assets/72544131/7f027dcb-9634-44c8-87e5-a31c09d5f870)
![image](https://github.com/bytedance/xgplayer/assets/72544131/005349ea-ea51-4b95-9087-b3988fb72fa5)

**Comment by amihhs** [comment]
首先我不太确定xgplayer-hls.js和xgplayer-hls的区别，文档也没有xgplayer-hls.js了。
如果你只是需要单独使用xgplayer-hls，[参考这里->独立使用场景](https://h5player.bytedance.com/plugins/extension/xgplayer-hls.html#独立使用场景)
